src/main/resources/docgen/templates/msmarco-doc-composite-tokenizer.template

```diff
@@ -0,0 +1,53 @@
+# Anserini Regressions: MS MARCO Document Ranking
```

Comment:
There is no corresponding yaml here? Don't understand what this template is for?

Reply:
Ah I see... So why just MS MARCO doc? Why not passage?

Do we want to circle back and do add composite {ms marco, dl19, dl20} x {passage, doc} in a separate PR?

Reply:
Okay, sure! I'll try to get it out over the weekend

Reply:
Do we need dl19 & dl20? I don't think we have those for `HuggingFaceTokenizerAnalyzer`